htdocs/modules/system/class/menu.php

```diff
@@ -176,7 +176,7 @@ public function render($currentoption = 1, $display = true)
             $menuItems[] = $menus;
         }
         $breadcrumb                = $menuItems[$currentoption];
-        $menuItems[$currentoption] = 'current';
+        /*$menuItems[$currentoption] = 'current';*/
```

Comment:
Actually, this is needed, as it identifies the current menu tab for highlighting.

Reply:
This and the DOM errors are cocvered in #1179

Thanks for pointing out the issue!